src/_common/store/app-store.ts

```diff
@@ -42,6 +48,19 @@ export class AppStore extends VuexStore<AppStore, Actions, Mutations> {
 		} else {
 			this.user = user;
 		}
+
+		if (user.timeout) {
+			const timeout = new UserTimeout(user.timeout);
+			if (timeout.isExpired) {
+				this.timeout = null;
```

Comment:
Would this ever happen? imo the backend should not return an expired timeout and we shouldn't care about time drifting.